ssolv-infrastructure/src/main/kotlin/org/depromeet/team3/place/client/GooglePlacesClient.kt

```
@@ -94,64 +113,88 @@ class GooglePlacesClient(
+                            span.setAttribute("http.protocol", response.version.toString())
+                            response.body<PlacesTextSearchResponse>().also { parsed ->
+                                span.setAttribute("google.places.result_count", (parsed.places?.size ?: 0).toLong())
+                            }
+                        }
+                    } catch (e: TimeoutCancellationException) {
+                        logger.error(e) { "Google Places API 텍스트 검색 타임아웃: query=$query" }
+                        throw PlaceSearchException(
+                            ErrorCode.PLACE_API_ERROR,
+                            detail = mapOf("query" to query, "error" to "요청 타임아웃 (${apiTimeoutMillis}ms 초과)")
+                        )
+                    } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)